core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -585,6 +588,12 @@ object AclCommand extends Logging {
       .describedAs("command-config")
       .ofType(classOf[String])
 
+    val loadAclCacheOpt = parser.accepts("load-acl-cache", "If you just run AclCommand,use this switch to avoid loading all ACL cache.")
```

Comment:
Can we not only load the cache if needed without an additional CLI flag?

Reply:
> Can we not only load the cache if needed without an additional CLI flag?
yes, you can.the loadAclCacheOpt is default "true"